### PR TITLE
Support for code-style CI

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Move badges behind the title.
- Add code-ci config files.

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
Ready for review.

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,551 @@
+[MASTER]
```
Comment (TomasTomecek) [inline .pylintrc]: Where did you get this file?

File: .landscape.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+strictness: medium
+
+max-line-length: 100
```
Comment (TomasTomecek) [inline .landscape.yaml]: nice, landscape can take care of it

File: .pylintrc (hunk 1 above)
Comment (lachmanfrantisek) [reply .pylintrc]: This is generated by pylint. This is the default, so we can tweak it in the future.

Comment (lachmanfrantisek) [reply .pylintrc]: ... and share between projects.

File: .pylintrc [hunk 3]
```diff
@@ -0,0 +1,551 @@
...
+known-third-party=enchant
+
+
+[DESIGN]
+
+# Maximum number of arguments for function / method
+max-args=5
+
+# Maximum number of attributes for a class (see R0902).
+max-attributes=7
```
Comment (jpopelka) [inline .pylintrc]: Not sure these two will be enough, but we'll see.

**Pull request merged**